executor/runtime/docker/docker.go

```diff
@@ -1408,9 +1401,30 @@ func (r *DockerRuntime) Start(parentCtx context.Context, pod *v1.Pod) (string, *
 		return "", nil, statusMessageChan, err
 	}
 
+	for _, c := range r.c.ExtraPlatformContainers() {
+		entry.Debugf("Adding status mmonitor for %s container (cid %s)", c.Name, c.ID)
+		eventChan, eventErrChan := r.getDockerEventsChannelsForContainer(eventCtx, c.ID)
+		go r.statusMonitor(eventCancel, c.ID, eventChan, eventErrChan, statusMessageChan)
```

Comment:
Shouldn't we keep track of these using something like a group?

Reply:
Not super sure. We don't want to wait for all monitors to finish, in fact we kinda want to fail as soon as any one of them finish and send along the status update.